roles/kubernetes/files/etc/systemd/system/kubelet.service.d/10-kubelet-common.conf

```diff
@@ -19,4 +21,5 @@ ExecStart=/usr/bin/kubelet \
     --cluster-dns $CLUSTER_DNS \
     --node-labels $DEFAULT_NODE_LABELS \
     --node-labels $EXTRA_NODE_LABELS \
-    $KUBELET_EXTRA_ARGS
\ No newline at end of file
+    --provider-id $PROVIDER_ID
```

Comment:
Can you add a continuation mark (`\`) here at the end of this line so the `$KUBELET_EXTRA_ARGS` don't get left out?

Reply:
done